app/assets/javascripts/markdown.js

```diff
@@ -49,6 +49,39 @@ $(() => {
     }
   });
 
+  $('.js-post-field').on('keydown', ev => {
+    if (ev.ctrlKey) {
+      switch (ev.keyCode) {
+        case 66:
+          $('[data-action="bold"]').click();
+          break;
+        
+        case 73:
+          $('[data-action="italic"]').click();
+          break;
+
+        case 75:
+          ev.preventDefault();
```

Comment:
Why are only some of these using `ev.preventDefault()`? Do the others work without it?

Reply:
yep that's right, specifically `ev.preventDefault()` is for `Ctrl+P` (opens the print dialog), `Ctrl+K` (focuses on the browser search bar), and `Ctrl+U` (opens the page source). The rest work without it.

Reply:
For the other cases, is preventDefault() a no-op or does it not work (because there is no default)?  If the former, I'm wondering if we should do it for all of them in case browsers add more overlapping key bindings in the future.  Disclosure: this is not my area, so if this is a dumb question please let me know.

Reply:
Ah, of course. I'm thinking of the default being "insert letter", but of course it's not if it's Ctrl. Thanks.